src/WinUI/ArcGIS.WinUI.Viewer/Samples/Map/SetBasemap/readme.metadata.json

```diff
@@ -0,0 +1,31 @@
+{
+    "category": "Map",
+    "description": "Change a map's basemap.",
+    "formal_name": "SetBasemap",
+    "ignore": false,
+    "images": [
+        "SetBasemap.jpg"
+    ],
+    "keywords": [
+        "basemap",
+        "basemap gallery",
+        "basemap style",
+        "map",
+        "toolkit"
+  ],
+    "offline_data": [],
+    "redirect_from": [
+        "/net/latest/winui/sample-code/set-basemap.htm"
```

Comment:
didn't we get rid of latest redirect links?
seeing this in README for all three platforms

Reply:
will remove this